src/main/java/org/opentripplanner/routing/algorithm/filterchain/ItineraryListFilterChainBuilder.java

```diff
@@ -45,28 +48,39 @@
     private DoubleFunction<Double> nonTransitGeneralizedCostLimit;
     private Instant latestDepartureTimeLimit = null;
     private Consumer<Itinerary> maxLimitReachedSubscriber;
-    private boolean reverseFilteringDirection;
 
 
-    /**
-     * @param arriveBy Used to set the correct sort order. This si the same flag as the
-     *        {@link org.opentripplanner.routing.api.request.RoutingRequest#arriveBy}.
-     */
-    public ItineraryListFilterChainBuilder(boolean arriveBy) {
-        this.arriveBy = arriveBy;
+    public ItineraryListFilterChainBuilder(SortOrder sortOrder) {
+        this.sortOrder = sortOrder;
     }
 
     /**
      * The maximum number of itineraries returned. This will remove all itineraries at the
      * end of the list AFTER the final sort of the itineraries.
      * <p>
+     * Se also the {@link #withMaxNumberOfItinerariesCrop(ListSection)} to change witch
```

Comment:
```suggestion
     * Se also the {@link #withMaxNumberOfItinerariesCrop(ListSection)} to change which
```

Reply:
I did a search an replace for all of the eval witches ;-)